### PR TITLE
formatted some output for main and workout session

### PR DESCRIPTION
**Pull request by yujinyang1998**

### CODE REVIEW

File: src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import ui.workout.workoutsession.WorkoutSessionUi;
+import workout.workoutsession.exercise.Exercise;
+
+import java.util.ArrayList;
+
+public class WorkoutSessionSearch extends Command {
+    @Override
+    public void execute(String[] input, ArrayList<Exercise> exercise,
+                        String filePath, Storage storage, boolean[] endWorkoutSession) {
+        WorkoutSessionUi.printHelp();
```
Comment (tienkhoa16) [inline src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java:14]: Printing help in workout session search does not seem so right :sweat_smile:

File: src/main/java/workout/workoutsession/WorkoutSession.java [hunk 2]
```diff
@@ -44,16 +43,19 @@ public void workoutSessionStart() {
...
             WorkoutSessionUi.printError();
         }
+
         while (!endWorkoutSession[0]) {
             try {
+                System.out.print("Workout Session >>>>> ");
                 workoutSessionProcessCommand(in.nextLine().trim());
             } catch (NullPointerException e) {
-                e.printStackTrace();
+                System.out.println("Please enter something.");
```
Comment (tienkhoa16) [inline src/main/java/workout/workoutsession/WorkoutSession.java:58]: Perhaps can implement the UI printing here also

**Review by tienkhoa16 — COMMENTED** [review]
LGTM 👍 Just a few points to note

File: src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java (hunk 1 above)
Comment (yujinyang1998) [reply src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java:14]: oh yeah I forgot to do this function

File: src/main/java/workout/workoutsession/WorkoutSession.java (hunk 2 above)
Comment (yujinyang1998) [reply src/main/java/workout/workoutsession/WorkoutSession.java:58]: will do

**Pull request merged**